src/scheduler/node_partition.c

```diff
@@ -364,7 +364,7 @@ find_node_partition_by_rank(node_partition **np_arr, int rank)
  * @retval	: NULL on error
  *
  */
-node_partition **
+node_partition**
 create_node_partition_by_partition(status *policy, node_info **nodes, char ** partitions, int *num_parts)
```

Comment:
Sorry for being confusing.  Leave the space between node_partition and **.  Remove the space in char ** partitions.

Reply:
With the change to support only one scheduler in place, this function isn't needed. I am removing the function itself.